e2e/content-services/upload/user-permission.e2e.ts

```diff
@@ -122,7 +122,7 @@ describe('Upload - User permission', () => {
             contentServicesPage.checkDragAndDropDIsDisplayed();
 
             let dragAndDrop = new DropActions();
-            let dragAndDropArea = element.all(by.css('adf-upload-drag-area div')).first();
+            let dragAndDropArea = contentServicesPage.dragAndDrop;
 
             dragAndDrop.dropFile(dragAndDropArea, emptyFile.location);
```

Comment:
I suggest creating methods dropFile and dropFolder inside ContentServicesPage, that will call the methods from DropActions.